includes/class-wcpdf-install.php

```diff
@@ -375,6 +380,13 @@ protected function upgrade( $installed_version ) {
 			@unlink( trailingslashit( $font_path ) . 'mustRead.html' );
 		}
 
+		// 2.12.1: schedule the yearly reset invoice number action
+		if ( version_compare( $installed_version, '2.12.1-dev-1', '<' ) ) {
```

Comment:
I don't understand this - run the code if the version is less than that version number.... the code comment doesn't really clarify why that is?

Reply:
This was done long time ago and I thought we could implement before the end of the year, but now I think we should postpone for the next year, not much time left, and maybe put your Updraft class in the equation.

Reply:
I will take a look next week.